course.yml

```diff
@@ -22,87 +22,8 @@ vars:
 plan:
 - title: "Představení, Jupyter notebook, základy pandas"
   slug: eda1
-  date: 2022-09-12
+  date: 2023-09-18
   materials:
   - lesson: pydata/install
   - lesson: pydata/notebook
   - lesson: pydata/pandas_core
-
-- title: "Datové typy a základy vizualizace v pandas"
-  slug: eda2
-  date: 2022-09-19
-  materials:
-  - lesson: pydata/pandas_types
-  - lesson: pydata/visualization_basics
-
-- title: "Explorativní analýza a statistika jedné proměnné"
-  slug: eda3
-  date: 2022-09-26
-  materials:
-  - lesson: pydata/eda-univariate-timeseries
-
-- title: "Explorativní datová analýza a více zdrojů / proměnných"
-  slug: eda4
-  date: 2022-10-03
-  materials:
-  - lesson: pydata/pandas_correlations
-
-- title: "Explorativní datová analýza a pokročilejší manipulace"
-  slug: eda5
-  date: 2022-10-10
-  materials:
-  - lesson: pydata/pandas_groupby
-
-- title: "Analýza hlavních komponent (PCA)"
-  slug: pca
-  date: 2022-10-24
-  materials:
-  - lesson: pydata/pca
-
-- title: "Úvod do strojového učení, regrese"
-  slug: ml1
-  date: 2022-10-31
-  materials:
-  - lesson: pydata/intro_regression
-  - lesson: pydata/regression_metrics
-  - lesson: pydata/regression_exercises
-  - lesson: pydata/regression_resume
-
-- title: "Knihovna Scikit-learn, ML workflow"
-  slug: ml2
-  date: 2022-11-07
-  materials:
-  - lesson: pydata/scikitlearn_api
-  - lesson: pydata/homework_revisited
-  - lesson: pydata/scikitlearn_resume
-
-- title: "Strojové učení, klasifikace"
-  slug: ml3
-  date: 2022-11-14
-  materials:
-  - lesson: pydata/intro_classification
-  - lesson: pydata/classification_metrics
-  - lesson: pydata/classification_resume
-
-- title: Interaktivní vizualizace a aplikace
-  slug: dashboards
-  date: 2022-11-21
-  materials:
-  - lesson: pydata/dashboards
-  
-- title: Databáze a SQL
-  slug: databases
-  date: 2022-11-28
-  materials:
-  - lesson: pydata/databases
-
-- title: API
-  slug: api
-  date: 2022-12-05
-  materials:
-  - lesson: pydata/api
-
-- title: "Svátky klidu a konce kurzu"
-  slug: konec
-  serial: null
-  date: 2022-12-24
```

Comment:
"Lekci" pro plánovaný konec kurzu doporučuju nechat, aby to po každé lekci nevypadalo že kurz už skončil.